x-pack/packages/kbn-ai-assistant/src/hooks/use_conversation.ts

```diff
@@ -106,8 +100,8 @@ export function useConversation({
         },
       })
       .catch((err) => {
-        notifications.toasts.addError(err, {
-          title: i18n.translate('xpack.observabilityAiAssistant.errorUpdatingConversation', {
+        notifications?.toasts.addError(err, {
```

Comment:
why is this now optional?

Reply:
These are all optional because we're using useKibana(), which only provides optional services. Observability plugins work around that by explicitly providing the services under `plugins.services` in Kibana's context, but that's an uncommon pattern across Kibana and I wanted to make as few assumptions as possible about which services are provided how. 

Hopefully we'll be able to remove these implicit dependencies and assumptions in future PRs.

Reply:
I would prefer this to be `!` instead of `?` which will just suppress errors, WDYT?